src/ui/mainwindow.cc

```diff
@@ -886,6 +880,10 @@ MainWindow::MainWindow( Config::Class & cfg_ ):
 #endif
 
   useSmallIconsInToolbarsTriggered();
+
+  if ( cfg.preferences.checkForNewReleases ) {
+    checkNewRelease();
+  }
```

Comment:
QTimer is better ,it is in the background.
current way will slow the startup for some time.

Reply:
Fixed.